webtop/__init__.py

```diff
@@ -133,63 +139,53 @@ def render_stats(stats: dict, _format: str) -> str:
         output = json.dumps(stats, indent=2)
     elif _format == "yaml":
         output = yaml.dump(stats, default_flow_style=False, sort_keys=False)  # type: ignore
+    # noinspection PyUnboundLocalVariable
```

Comment:
This line doesn't seem to trigger an error in CI. Remove

Reply:
Ahh, that was to make the Pycharm linter be quiet. I'll yank that out.